stylesheets/styles.css

```diff
@@ -331,13 +337,6 @@ pr-badge a:focus::after  {
 }
 
 /* Min-width break to add more padding on wider screens */
-@media all and (max-width: 720px) {
-	#sidebar {
-		display: none;
-	}
-}
-
-/* Min-width break to add more padding on wider screens */
```

Comment:
@dakebl By default it has the class "closed" in index.html, so it will start out hidden.